### DISCUSSION TITLE
Настройка общего поиска

### DISCUSSION THREAD

**Discussion by smplseven** [totumonline/totum-ru-issues-and-discussions#279] [Как это сделать?] [answered]
Сейчас общий поиск в системе работает по первым символам введенных значений. т.е. при вводе "5566" не покажет значения 115566, но покажет "55667" и "55668". Мы столкнулись с кейсом, когда нужно в поиске отображать и данные не сначала строки.

Возможно ли включить нестрогий поиск и уменьшить количество символов для поиска до 4х?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Can meilisearch search a part of words?

MeiliSearch is a prefix-based search engine and therefore cannot find parts of the query that lie in the middle or at the end of a word.

Насчет количества символов для срабатывания квери посмотрим позже.

**Reply by smplseven** [reply]
Спасибо!

**Reply by ttm-alexeypolunin** [reply]
Относительно поиска части. Вы можете при генерации поля search сгенерировать текс с кусками (если куски известны заранее). Те если у вас номер чего-то 46027778798 и в нем 4602 это префикс и 777 и 8798 номера документов и поиск нужен именно по номерам. То в текст засунте отдельно что-то типа [Теги: 777, 8798]

**Reply by smplseven** [reply]
Да,как раз планировали так сделать,спасибо!
